multimodal/tests/unittests/test_data_augmentation.py

```diff
@@ -51,6 +51,7 @@ def test_mixup():
     hyperparameters = {
         "optimization.max_epochs": 1,
         "optimization.top_k_average_method": BEST,
+        "model.t_few.checkpoint_name": "t5-small",
```

Comment:
Is the previous test still runnable without this line?

Reply:
The default t_few model is the 3B one, so the CI cannot handle it. Is that addressing your point?